astropy/io/misc/asdf/schemas/astropy.org/astropy/time/timedelta-1.0.0.yaml

```diff
@@ -0,0 +1,30 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://astropy.org/schemas/astropy/time/time-1.1.0"
+title: Represents an instance of TimeDelta from astropy
+description: |
+  Represents the time difference between two times.
+
+type: object
+properties:
+    jd1:
+      type: number
+      description: |
+        Value representing first 64 bits of precision
+    jd2:
+      type: number
+      description: |
+        Value representing second 64 bits of precision
+    format:
+      type: string
+      description: |
+        Format of time value representation.
```

Comment:
Should this be an enum too? At the moment, it can only be `[sec, jd, datetime]`.

Reply:
There was originally an `enum` here, but @Cadair left a comment indicating that this might not be a good idea: https://github.com/astropy/astropy/pull/8285#pullrequestreview-185237945

Reply:
Yes, @Cadair is completely right. We should not have an enum. (Indeed, there was some talk having provisions for adding new scales - but let's cross that bridge when we get there!)